lib/mpl_toolkits/mplot3d/art3d.py

```diff
@@ -329,6 +350,16 @@ def __init__(self, *args, zs=(), zdir='z', **kwargs):
         self.set_3d_properties(zs, zdir)
 
     def set_3d_properties(self, verts, zs=0, zdir='z'):
```

Comment:
Document `verts`?

Reply:
If one look two lines up, this is not called using `verts`. Will try to figure out the correct behavior, but since that is a bug-fix I'll keep it outside of this PR.